core/src/main/java/io/grpc/internal/DelayedStream.java

```diff
@@ -94,6 +98,22 @@ public void run() {
     });
   }
 
+  @Override
+  public void appendTimeoutInsight(InsightBuilder insight) {
+    synchronized (this) {
+      if (listener == null) {
+        return;
+      }
+      if (realStream != null) {
+        insight.appendKeyValue("buffered_nanos", streamSetTimeNanos - startTimeNanos);
+        realStream.appendTimeoutInsight(insight);
+      } else {
+        insight.append("waiting_for_connection");
```

Comment:
Nit: it'd be nice if these cases were more symmetric in the output string, moving this to the next line.

Reply:
Done.